packages/cli/src/commands/inspect/index.ts

```diff
@@ -192,6 +192,23 @@ async function printDetails({
   print(chalk.bold('  General\n\n'));
   print(`    ${chalk.cyan('id')}\t\t${id}\n`);
   print(`    ${chalk.cyan('name')}\t${name}\n`);
+  // @ts-ignore - customEnvironment needs to be defined on Deployment
+  const customEnvironmentName = deployment.customEnvironment?.name;
+  const target = customEnvironmentName ?? deployment.target;
```

Comment:
`deployment.target` has type `"staging" | "production" | "null" | "undefined"`. Even after spending time digging into this, I'm still unsure what "null" or "undefined" mean.

If the deployment doesn't have a custom environment name, and `target` is `null` or `undefined`, should it default to displaying something?

Reply:
I think displaying nothing is fine here. Pretty edge-casey. 

👋 hello future someone reading this wondering why we didn't handle this case!

Reply:
`null` means "preview", so we should special-case that. `undefined` I think does not ever happen.